docker/README.md

```
@@ -100,56 +100,48 @@ This can be useful during development to:
-   docker compose logs -f [service_name]
-   ```
+Enable debugpy by adding `compose.debug.yaml`:
+
+```bash
+VERSION=dev docker compose -f docker-compose.yaml -f compose.override.yaml -f compose.debug.yaml watch
+```
+
+Debug ports per service:
+
+| Service | Port |
+|---------|------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)